### PR TITLE
Feature/cc 0083 admin tile type page

### PR DESCRIPTION
**Pull request by GitHubEason**

### CODE REVIEW

File: src/components/SideBar/components/SideBarItem/SideBarItem.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        _hover={{ bg: "button.hover" }}
+        color="fontcolor.primary"
+      >
+        <Icon w="32px" h="32px" viewBox="0 0 20 20" paddingRight="5px">
+          {e.icon}
+        </Icon>
+        <Link href={e.href}>
+          <a>{e.title}</a>
+        </Link>
+      </Box>
```
Comment (tatoMa) [inline src/components/SideBar/components/SideBarItem/SideBarItem.tsx:21]: Using link wrapping the whole tag will make it a big clickable button.

File: next-env.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-// / <reference types="next" />
-// / <reference types="next/image-types/global" />
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
 // see https://nextjs.org/docs/basic-features/typescript for more information.
```
Comment (Fuchih) [inline next-env.d.ts:5]: put this file to gitignore

File: src/components/SideBar/components/SideBarItem/SideBarItem.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    );
+  });
+};
```
Comment (Fuchih) [inline src/components/SideBar/components/SideBarItem/SideBarItem.tsx:24]: typically e stands for an event, maybe choose another word?

File: src/components/SideBar/SideBar.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      <Grid
+        templateAreas={`"logo"
+                  "sideBarItem"
+                  "divider"
+                  " Profile"`}
+        gridTemplateRows={"10vh 70vh 2vh"}
+        fontWeight="bold"
+        margin="20px"
+      >
+        <GridItem pl="2" area={"logo"}>
```
Comment (Fuchih) [inline src/components/SideBar/SideBar.tsx:18]: pl --> paddingLeft

File: src/components/SideBar/components/Logo/Logo.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { Text, Flex } from "@chakra-ui/react";
+import DashBoardLogo from "@/assets/svg/SideBarSvg/dashboard-log.svg";
+const Logo = () => {
+  return (
+    <Flex paddingTop="20px">
+      <DashBoardLogo />
+      <Text
+        m={2}
```
Comment (Fuchih) [inline src/components/SideBar/components/Logo/Logo.tsx:8]: m --> margin

File: src/components/SideBar/components/Profile/Profile.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import { Box, Text, Flex, Icon } from "@chakra-ui/react";
+import { BsFillFilePersonFill } from "react-icons/bs";
+import { MdOutlineLogout } from "react-icons/md";
+const Profile = () => {
+  return (
+    <Box>
+      <Text color="fontcolor.primary">PROFILE</Text>
+
+      <Flex paddingTop="20px" color="fontcolor.primary">
+        <Icon w="32px" h="32px" viewBox="0 0 20 22" paddingRight="5px">
```
Comment (Fuchih) [inline src/components/SideBar/components/Profile/Profile.tsx:10]: 1. w --> width
2. h --> height

File: src/components/SideBar/components/SideBarItem/SideBarItem.tsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+      <Box
+        key={e.id}
+        width="full"
+        borderBottom="1px"
+        padding="15px 0px"
+        _hover={{ bg: "button.hover" }}
+        color="fontcolor.primary"
+      >
+        <Icon w="32px" h="32px" viewBox="0 0 20 20" paddingRight="5px">
```
Comment (Fuchih) [inline src/components/SideBar/components/SideBarItem/SideBarItem.tsx:15]: 1. w --> width
2. h --> height

File: src/layouts/Header.tsx [hunk 8]
```diff
@@ -16,7 +16,13 @@ const Header = () => {
...
       </Head>
-      <NextHeadSeo title="CourtCanva" canonical="" description="" customMetaTags={[]} og={{}} />
+      <NextHeadSeo
+        title="CourtCanva Admin"
+        canonical=""
+        description=""
+        customMetaTags={[]}
+        og={{}}
+      />
     </>
```
Comment (Fuchih) [inline src/layouts/Header.tsx:26]: do we need SEO for CMS?

Comment (tatoMa) [reply src/layouts/Header.tsx:26]: No, we do not need SEO. I guess it's from the old cc-app project starter.

**Comment by GitHubEason** [comment]
Screen Shot 2022-07-11 at 1 25 42 pm
show tiles details 
Screen Shot 2022-07-11 at 1 26 10 pm
admin can edit tile

**Review by EmilyGong111 — COMMENTED** [review]
The dotenv file was not ignored.

**Comment by Jzeva** [comment]
image

Do you think we need to add some validation for color input? Because when I tested creating new tile, I found that I could add a color called "123" and actually it is not a color code and cc-app wont make it as a color when users try to customize the court. Besides, I tried to enter negative number for height and length, it won't be stored into the database. But I didn't get any warning informing me of that. I think it is necessary to validate the data input in the front-end as well.

**Review by Fuchih — APPROVED** [review]

**Pull request closed without merging**